test/vm.install

```diff
@@ -43,6 +43,10 @@ ln -s /usr/local/bin/node /usr/local/bin/nodejs
 if type firewall-cmd >/dev/null 2>&1; then
     firewall-cmd --add-service=cockpit --permanent
 fi
+
+# mock compose creation calls to backend for testing purposes
+sed -i "s|\"/api/v0/compose\"|\"/api/v0/compose?test=2\"|" /usr/share/cockpit/welder/dist/main.*.js
```

Comment:
In principle this approach is better indeed, as long as you can actually rely on the .js files from the installed rpm. When you build in production mode, these are usually compressed and minified, so that sed wouldn't work right away.

But why wouldl that not work when running the test locally? It seems to work fine. So the commit message should be clearer about the rationale.